### DISCUSSION TITLE
Welcome to Cantoboard Discussions!

### DISCUSSION THREAD

**Discussion by alex-the-man** [Cantoboard/Cantoboard#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 4705-SPWNE** [comment]
you deserve to be in your country, traitor.
